### DISCUSSION TITLE
sRGB gamma exponent in apca-w3

### DISCUSSION THREAD

**Discussion by jengelh** [Myndex/apca-w3#16] [General]
The current apca-w3 code at https://github.com/Myndex/apca-w3/blob/c012257167d822f91bc417120bdb82e1b854b4a4/src/apca-w3.js#L148 employs a trivial 2.4 curve:

```js
  function simpleExp (chan) { return Math.pow(chan/255.0, SA98G.mainTRC); };
```

Normally though, sRGB is described as having 2.2.
The value 2.4 is a consequence of sRGB [reportedly defining](https://mina86.com/2019/srgb-xyz-conversion/) a linear section for pixel values 0..10 that necessitates a harder (2.4) curvature for pixel values 11..255 to get the combined curve to approimxate a 2.2:

```c
	return value255 <= 10
		? value255 / 3294.6
		: Math.pow((value255 + 14.025) / 269.025, 2.4);
```

Should apca-w3 switch from the trivial 2.4 curve to a trivial 2.2 curve, or use the {linear + 2.4 segment} composed curve? What's your take?

**Comment by Myndex** [comment]
Hi Jan @jengelh  and thank you for your question. 

### Short Answer
The short answer is this is intentional pre-shaping, and possibly going to change before release.

### Additional
This is answered more fully in the main documentation, [**"regardingexponents.md"**](https://github.com/Myndex/SAPC-APCA/blob/master/documentation/regardingexponents.md) at the documentation repo.

Also, the main discussion area regarding APCA and related color theory is at the SAPC-APCA repo:

[**github.com/Myndex/SAPC-APCA/discussions**](https://github.com/Myndex/SAPC-APCA/discussions)

If you'd like to discuss or explore this more, let me know and I'll re-open this thread and move it to the main area.
